docs/src/quickstart.md

```diff
@@ -1,23 +1,17 @@
 # Quickstart
 
-This section describes how to test a PostgreSQL cluster on your laptop/computer
-using CloudNativePG on a local Kubernetes cluster in [Kind](https://kind.sigs.k8s.io/) or
+This section guides you through testing a PostgreSQL cluster on your local machine by deploying CloudNativePG on a local Kubernetes cluster using either [Kind](https://kind.sigs.k8s.io/) or
 [Minikube](https://kubernetes.io/docs/setup/learning-environment/minikube/).
 
-!!! Warning
-    The instructions contained in this section are for demonstration,
-    testing, and practice purposes only and must not be used in production.
+!!! Warning The instructions contained in this section are for demonstration, testing, and practice purposes only and must not be used in production.
 
 Like any other Kubernetes application, CloudNativePG is deployed using
 regular manifests written in YAML.
 
 By following the instructions on this page you should be able to start a PostgreSQL
 cluster on your local Kubernetes installation and experiment with it.
 
-!!! Important
-    Make sure that you have `kubectl` installed on your machine in order
-    to connect to the Kubernetes cluster. Please follow the Kubernetes documentation
-    on [how to install `kubectl`](https://kubernetes.io/docs/tasks/tools/install-kubectl/).
+!!! Important Make sure that you have `kubectl` installed on your machine in order to connect to the Kubernetes cluster. Please follow the Kubernetes documentation on [how to install `kubectl`](https://kubernetes.io/docs/tasks/tools/install-kubectl/).
```

Comment:
Please undo this and all the changes that alter the layout of admonition boxes.
They're the way they are for a reason.